ui/src/components/pages/Pipeline/partials/Graph/index.tsx

```diff
@@ -175,7 +175,7 @@ const Flow = (props: FlowProps) => {
   );
   const [statusPayload, setStatusPayload] = useState<any>(undefined);
   const [timerDateStamp, setTimerDateStamp] = useState<any>(undefined);
-  const [timer, setTimer] = useState<number | undefined>(undefined);
```

Comment:
Why is this any, because timer will return number and when it is cleared it will be undefined

Reply:
had set it as any because, we assign a pauseTimer variable of type `NodeJS.Timer` inside `handleTimer` function

Reply:
so either we can add `NodeJS.Timer` in addition to the above two or keep it as `any`